Frontend/CO.CDP.OrganisationApp/Pages/Supplier/SupplierWebsiteQuestion.cshtml.cs

```diff
@@ -49,6 +48,22 @@ public async Task<IActionResult> OnGet(Guid id)
 
     public async Task<IActionResult> OnPost()
     {
+        if (!string.IsNullOrWhiteSpace(WebsiteAddress))
+        {
+            WebsiteAddress = WebsiteAddress.Trim();
+
+            if (!WebsiteAddress.StartsWith("http") && !WebsiteAddress.StartsWith("//"))
+            {
+                WebsiteAddress = $"https://{WebsiteAddress}";
```

Comment:
Isn't it safer to prepend `http`? It usually redirects to https, while if https is missing for the website it just won't work.

Reply:
good idea, made the change